shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IConfiguration.ts

```diff
@@ -176,4 +176,11 @@ export interface IConfiguration {
      * this will not send any notifications.
      */
     disableDbgExt?: boolean;
+
+
+     /**
+     * Custom optional value that will be added as a prefix for storage name.
+     * @defaultValue undefined
+     */
+     storagePrefix?:string;
```

Comment:
Just realized that we also need to add this configuration item to the main READMD.md for documentation purposes describing why and when this option should be used.

Reply:
I inserted in the last line of configuration table in main readmd https://github.com/microsoft/ApplicationInsights-JS#icookiemgrconfig, do we need adding more info?